CONTRIBUTING.md

```diff
@@ -1,97 +1,31 @@
-# Contributing to Flutter
+# Contributing to Flutter Packages
 
-[![Build Status](https://api.cirrus-ci.com/github/flutter/packages.svg)](https://cirrus-ci.com/github/flutter/packages)
+[![Build Status](https://api.cirrus-ci.com/github/flutter/packages.svg)](https://cirrus-ci.com/github/flutter/packages/master)
 
-_See also: [Flutter's code of conduct](https://flutter.io/design-principles/#code-of-conduct)_
+_See also: [Flutter's code of conduct](https://github.com/flutter/flutter/blob/master/CODE_OF_CONDUCT.md)_
 
-## Things you will need
+## Welcome
 
-- Linux, Mac OS X, or Windows.
-- git (used for source version control).
-- An ssh client (used to authenticate with GitHub).
+For an introduction to contributing to Flutter, see [our contributor
```

Comment:
I'd add a line here encouraging submissions. The section is called "Welcome", afterall. :-)

Reply:
I was matching flutter/engine (and now flutter/plugins) here, which leaves the real welcome text to the main contributing doc that it's liking to.

Reply:
Yeah, I don't mean put in all the details, just literally "We gladly accept contributions via GitHub pull requests." or something.

Reply:
My main reason for not wanting to add it here is consistency with the other satellite repos, but I think it's also nice that the main repo's welcome acknowledges that there are many ways someone could contribute, rather than singling out PRs. And avoiding that here would mean copying a bunch more text, which gets into duplication...